frontend/src/pages/NicknamePage/index.tsx

```diff
@@ -17,9 +17,8 @@ type Props = {
 const NicknamePage = ({ gameCode }: Props) => {
```

Comment:
Should add `const { setMe } = useContext(PlayersContext) to this component`

Reply:
This is no longer required as all player IDs are returned and the current player has their playerId stored in persistent state